samples/SpectatorView.Example.Unity/README.md

```diff
@@ -1,3 +1,70 @@
-# SpectatorView General Sample
+# SpectatorView.Example.Unity
 
-This sample exercises many aspects of SpectatorView with QR Code localization technique. For instructions on configuring, building and deploying please refer to the main [setup instructions](../../doc/SpectatorView.Setup.md).
\ No newline at end of file
+This sample shows how to use [QR Code detection](https://docs.microsoft.com/en-us/windows/mixed-reality/qr-code-tracking) to spatially align devices in the physical world.
+
+## Running the Demo
+
+In order to run the demo, you will need a HoloLens and an AR capable mobile device. Once you have the devices ready, follow the instructions below.
+
+> **ARKit Note:** If you wish to run the experience on an iOS device with ARKit, download the [ARKit repository](https://bitbucket.org/Unity-Technologies/unity-arkit-plugin/downloads/), unzip it, and copy the contents of the `Unity-Technologies-unity-arkit-plugin-94e47eae5954/Assets` folder to `/samples/Build2019Demo.Unity/Assets/`.
+
+### Instructions
+
+These instructions assume that a HoloLens device will be used to host the shared experience and that an Android phone will be used to spectate the shared experience. For iOS, replace references to Android with iOS.
```

Comment:
"For iOS, replace references to Android with iOS"

This isn't exactly true right? Below you also have iOS instructions. In fact there doesn't appear to be any special treatment of Android aside from the fact that it's sequentially above iOS.

Reply:
removed

---
In reply to: [352945123](https://github.com/microsoft/MixedReality-SpectatorView/pull/290#discussion_r352945123) [](ancestors = 352945123)